src/Symfony/Component/Form/Tests/Extension/Core/Type/CheckboxTypeTest.php

```diff
@@ -170,4 +170,38 @@ public function provideCustomModelTransformerData()
             array('unchecked', false),
         );
     }
+
+    public function testSubmitNullWithEmptyDataTrueWhenNested()
+    {
+        $form = $this->factory->create('form')
```

Comment:
Neither this one nor `testSubmitNullWithEmptyData`(which is the same without nesting) do pass without the change.